config.yml

```diff
@@ -0,0 +1,323 @@
+  - name: RBS::AST::Members::Alias
+    fields:
+      - name: new_name
+      - name: old_name
+      - name: kind
+      - name: annotations
+      - name: location
+      - name: comment
+  - name: RBS::AST::Members::AttrAccessor
+    fields:
+      - name: name
+      - name: type
+      - name: ivar_name
+      - name: kind
+      - name: annotations
+      - name: location
+      - name: comment
+      - name: visibility
+  - name: RBS::AST::Members::AttrReader
+    fields:
+      - name: name
+      - name: type
+      - name: ivar_name
+      - name: kind
+      - name: annotations
+      - name: location
+      - name: comment
+      - name: visibility
+  - name: RBS::AST::Members::AttrWriter
+    fields:
+      - name: name
+      - name: type
+      - name: ivar_name
+      - name: kind
+      - name: annotations
+      - name: location
+      - name: comment
+      - name: visibility
+  - name: RBS::AST::Members::ClassInstanceVariable
+    fields:
+      - name: name
+      - name: type
+      - name: location
+      - name: comment
+  - name: RBS::AST::Members::ClassVariable
+    fields:
+      - name: name
+      - name: type
+      - name: location
+      - name: comment
+  - name: RBS::AST::Members::Extend
+    fields:
+      - name: name
+      - name: args
+      - name: annotations
+      - name: location
+      - name: comment
+  - name: RBS::AST::Members::Include
+    fields:
+      - name: name
+      - name: args
+      - name: annotations
+      - name: location
+      - name: comment
+  - name: RBS::AST::Members::InstanceVariable
+    fields:
+      - name: name
+      - name: type
+      - name: location
+      - name: comment
+  - name: RBS::AST::Members::MethodDefinition
+    fields:
+      - name: name
+      - name: kind
+      - name: overloads
+      - name: annotations
+      - name: location
+      - name: comment
+      - name: overloading
+      - name: visibility
+  - name: RBS::AST::Members::MethodDefinition::Overload
+    fields:
+      - name: annotations
+      - name: method_type
+  - name: RBS::AST::Members::Prepend
+    fields:
+      - name: name
+      - name: args
+      - name: annotations
+      - name: location
+      - name: comment
+  - name: RBS::AST::Members::Private
+    fields:
+      - name: location
+  - name: RBS::AST::Members::Public
+    fields:
+      - name: location
+  - name: RBS::AST::TypeParam
+    fields:
+      - name: name
+      - name: variance
+      - name: upper_bound
+      - name: default_type
+      - name: location
+  - name: RBS::MethodType
+    fields:
+      - name: type_params
+      - name: type
+      - name: block
+      - name: location
+  - name: RBS::Namespace
+    fields:
+      - name: path
+      - name: absolute
+  - name: RBS::TypeName
+    fields:
+      - name: namespace
+      - name: name
+  - name: RBS::Types::Alias
+    fields:
+      - name: name
+        c_name: typename # Temporary name to match existing C code. Will be removed in next PR.
```

Comment:
The identifiers used in C don't always match those used in the Ruby API. Where necessary, we used this c_name: field to preserve the C code as it was.